pkg/storage/enginepb/engine.go

```diff
@@ -43,7 +43,7 @@ func (e *EngineType) Set(s string) error {
 		*e = EngineTypePebble
 	default:
 		return fmt.Errorf("invalid storage engine: %s "+
-			"(possible values: rocksdb, pebble)", s)
+			"(possible values: pebble)", s)
```

Comment:
@itsbilal @sumeerbhola - is there any reason to keep this code around now that we only support a single storage engine? I noticed that it was left as part of the cleanup in #55509, so wasn't sure if there was a particular reason.